pkgs/development/python-modules/autobahn/default.nix

```diff
@@ -49,22 +49,14 @@
 
 buildPythonPackage rec {
   pname = "autobahn";
-  version = "22.7.1";
+  version = "23.6.2";
   format = "setuptools";
 
-  disabled = pythonOlder "3.7";
-
-  patches = [
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
done